browser-test/src/admin_predicates.test.ts

```diff
@@ -295,8 +301,18 @@ describe('create and edit predicates', () => {
       '100.01',
     )
 
-    // Date predicate
+    // Date predicate is on or before
     await adminPrograms.goToEditBlockPredicatePage(programName, 'Screen 7')
+    await adminPredicates.addPredicate(
+      'predicate-date',
```

Comment:
This need to be the new question name you're making.

Currently you're using the same question in both predicates.